pydantic_settings/main.py

```diff
@@ -120,11 +137,13 @@ def prepare_field(cls, field: ModelField) -> None:
         @classmethod
         def customise_sources(
             cls,
-            init_settings: SettingsSourceCallable,
-            env_settings: SettingsSourceCallable,
-            file_secret_settings: SettingsSourceCallable,
-        ) -> Tuple[SettingsSourceCallable, ...]:
-            return init_settings, env_settings, file_secret_settings
+            settings_cls: Type['BaseSettings'],
```

Comment:
either now or in a separate PR, let's add `from __future__ import annotations as _annotations` so these don't need to be strings.

Reply:
Done